sites/rcvfortexas-1/donate.html

```diff
@@ -0,0 +1,391 @@
+<div id="content">
+
+  {% if page.headline.size > 0 %}
+  	<div id="headline">
+  	  <h2>{{ page.headline }}</h2>
+  	</div>
+  {% endif %}
+  <p class="donate-type-intro">Use my donation for:</p>
+  <div class="donate-type-selector">
+    <div>
+      <div class="donate-type-button active">
+        <div class="donate-type-heading">Education &amp; Outreach</div>
+        <div>Tax-deductible</div>
+        <div>501(c)(3)</div>
+      </div>
+    </div>
+    <div>
+      <a class="donate-link-noshadow" href="/donate_3">
+      <div class="donate-type-button">
+        <div class="donate-type-heading">Legislative Advocacy</div>
+        <div>Not tax-deductible</div>
+        <div>501(c)(4)</div>
+      </div>
+      </a>
+    </div>
+  </div>
+  {% if page.donation_v2.content.size > 0 %}
+  <div id="intro" class="intro">
+    {{ page.donation_v2.content }}
+  </div>
+  {% endif %}
+  
+ {% assign thermo_width = 100 %}
+
+    {% if page.donation_v2.has_amount_goal? %}
+    <div class="clearfix">
+      {% if page.donations_count < 1 %}
+      <div class="bar-text bar-text-null">JUST STARTED</div>
+      {% else %}
+      <div class="bar-text">{{ page.donations_amount_format }} raised</div>
+      {% endif %}
+      <div class="bar-goal">GOAL: {{ page.donation_v2.amount_goal_format }}</div>
+      <div class="progress" style="width: {{ thermo_width }}%;">
+        <div class="bar bar-success" style="width: {{ page.donation_v2.percent_of_amount_goal | times:100 }}%;">
+        </div>
+      </div>
+    </div>
+
+    {% elsif page.donation_v2.has_donor_goal? %}
+
+    <div class="clearfix">
+      {% if page.donors_count < 2 %}
+      <div class="bar-text bar-text-null">JUST STARTED</div>
+      {% else %}
+      <div class="bar-text">{{ page.donors_count }} donors</div>
+      {% endif %}
+      <div class="bar-goal">{{ page.donation_v2.donor_goal }} donors</div>
+      <div class="progress" style="width: {{ thermo_width }}%;">
+        <div class="bar bar-success" style="width: {{ page.donation_v2.percent_of_donor_goal | times:100 }}%;">
+        </div>
+      </div>
+    </div>
+
+    {% endif %}
+    
+  {% form_for donation %}
+
+  <div class="form-wrap">
+    <div class="form">
+
+      <div class="form-errors">
+        {% error_messages_for donation %}
+      </div>
```

Comment:
Again, this code is fine, but when I scrolled down enough to try submit the form with errors, I couldn't see the error messages. It might be nice to move this down to just above or below the submit button.